rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -564,18 +580,34 @@ bool Z3ConvVisitor::VisitBinaryOperator(clang::BinaryOperator *c_op) {
       InsertZ3Expr(c_op, lhs & rhs);
       break;
 
+    case clang::BO_Or:
+      InsertZ3Expr(c_op, lhs | rhs);
+      break;
+
     case clang::BO_Xor:
       InsertZ3Expr(c_op, lhs ^ rhs);
       break;
 
+    case clang::BO_Mul:
+      InsertZ3Expr(c_op, lhs * rhs);
+      break;
+
+    case clang::BO_Div:
```

Comment:
See above comment on remainders. `z3::udiv()` and `z3::sdiv()` should be used for signed and unsigned operations, respectively.